dd_game.yml

```diff
@@ -0,0 +1,184 @@
+  - pycodestyle=2.7.0=pyhd3eb1b0_0
+  - pyflakes=2.3.1=pyhd3eb1b0_0
+  - pygments=2.10.0=pyhd3eb1b0_0
+  - pyparsing=2.4.7=pyhd3eb1b0_0
+  - pytest=6.1.2=py38h06a4308_0
+  - python=3.8.5=h7579374_1
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python_abi=3.8=2_cp38
+  - pytz=2021.3=pyhd3eb1b0_0
+  - rdkit=2020.09.3=py38h3f7302f_0
+  - readline=8.1=h27cfd23_0
+  - reportlab=3.5.68=py38hadf75a6_0
+  - setuptools=58.0.4=py38h06a4308_0
+  - sip=4.19.13=py38he6710b0_0
+  - six=1.16.0=pyhd3eb1b0_0
+  - sqlalchemy=1.4.22=py38h497a2fe_0
+  - sqlite=3.36.0=hc218d9a_0
+  - tk=8.6.11=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tornado=6.0.4=py38h1e0a361_2
+  - traitlets=5.1.0=pyhd3eb1b0_0
+  - wcwidth=0.2.5=pyhd3eb1b0_0
+  - wheel=0.37.0=pyhd3eb1b0_1
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.0.10=h7f98852_0
+  - xorg-libsm=1.2.2=h470a237_5
+  - xorg-libx11=1.7.2=h7f98852_0
+  - xorg-libxext=1.3.4=h7f98852_1
+  - xorg-libxrender=0.9.10=h7f98852_1003
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xz=5.2.5=h7b6447c_0
+  - zipp=3.6.0=pyhd3eb1b0_0
+  - zlib=1.2.11=h7b6447c_3
+  - zstd=1.4.9=haebb681_0
+  - pip:
+    - altair==4.1.0
+    - arcade==2.5.6
+    - argon2-cffi==21.1.0
+    - astor==0.8.1
+    - backports-zoneinfo==0.2.1
+    - base58==2.1.0
+    - blinker==1.4
+    - cachetools==4.2.4
+    - cffi==1.15.0
+    - charset-normalizer==2.0.7
+    - click==7.1.2
+    - debugpy==1.5.1
+    - defusedxml==0.7.1
+    - entrypoints==0.3
+    - flask==2.0.2
+    - flask-cors==3.0.10
+    - gitdb==4.0.9
+    - gitpython==3.1.24
+    - idna==3.3
+    - ipykernel==6.4.2
+    - ipython==7.28.0
+    - ipython-genutils==0.2.0
+    - ipywidgets==7.6.5
+    - itsdangerous==2.0.1
+    - jinja2==3.0.2
+    - jsonschema==4.1.2
+    - jupyter-client==7.0.6
+    - jupyter-core==4.8.1
+    - jupyterlab-pygments==0.1.2
+    - jupyterlab-widgets==1.0.2
+    - markupsafe==2.0.1
+    - matplotlib-inline==0.1.3
+    - mistune==0.8.4
+    - mols2grid==0.1.0
+    - nbclient==0.5.4
+    - nbconvert==6.2.0
+    - nbformat==5.1.3
+    - nest-asyncio==1.5.1
+    - notebook==6.4.5
+    - numpy==1.20.1
+    - pandocfilters==1.5.0
+    - pillow==8.2.0
+    - prometheus-client==0.11.0
+    - protobuf==3.19.0
+    - pyarrow==5.0.0
+    - pycparser==2.20
+    - pydeck==0.7.1
+    - pyglet==1.5.21
+    - pymunk==6.0.0
+    - pyrsistent==0.18.0
+    - python-graphviz==0.17
+    - pytiled-parser==0.9.4a3
+    - pytz-deprecation-shim==0.1.0.post0
+    - pyyaml==5.4.1
+    - pyzmq==22.3.0
+    - requests==2.26.0
+    - send2trash==1.8.0
+    - shapely==1.7.1
+    - smmap==5.0.0
+    - streamlit==1.1.0
+    - terminado==0.12.1
+    - testpath==0.5.0
+    - toolz==0.11.1
+    - tzdata==2021.5
+    - tzlocal==4.0.1
+    - urllib3==1.26.7
+    - validators==0.18.2
+    - watchdog==2.1.6
+    - werkzeug==2.0.2
+    - widgetsnbextension==3.5.1
+prefix: /home/sabsr3/miniconda3/envs/dd_game
```

Comment:
Is this your file location - should this be here?

Reply:
good catch, @DylanAdlard did this work fro you when you installed from requirements? Bc if yes then I guess we can leave it for now?